lib/util/lifted/influx/coordinator/statement_executor.go

```diff
@@ -1074,7 +1074,7 @@ func (e *StatementExecutor) executeSelectStatement(stmt *influxql.SelectStatemen
 		qStat, _ := ctx.Value(query2.QueryDurationKey).(*statistics.SQLSlowQueryStatistics)
 		if qStat != nil {
 			qStat.AddDuration("SqlIteratorDuration", end.Sub(start).Nanoseconds())
-			qStat.AddDuration("EmitDuration", time.Now().Sub(end).Nanoseconds())
+			qStat.AddDuration("EmitDuration", time.Since(end).Nanoseconds())
```

Comment:
In the defer function，can you change the "end.Sub(start).Nanoseconds()" to “end.Since(start).nanoseconds”？or all use “time.Since(xxx)” instead

Reply:
There's not an  `end.Since()` function, that's it is not possible to make the required changes.

`time.Since()` is a shortcut for `time.Now().Sub()` with performance optimization, so it is recommend to replace the latter with the former.

Reply:
sorry， I read `end.Sub(start)` wrong.